config.json

```
@@ -198,6 +198,16 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 1,
+        "topics": null,
+        "deprecated": true
```

```suggestion
        "status": "deprecated"
```

See https://exercism.org/docs/building/tracks/deprecated-exercises

---

Sorry, did look at an [old PR](https://github.com/exercism/nim/pull/155/files) and didn't check if that configuration was still up to date.